src/Tribe/Customizer/Section.php

```diff
@@ -206,7 +206,7 @@ public function filter_settings( $settings, $search ) {
 			return $settings;
 		}
 
-		if ( count( $search ) === 1 ) {
+		if ( is_array( $search ) && count( $search ) === 1 ) {
```

Comment:
I think this one should be fine but we can always cast the variable as well.

Reply:
Hmm, but if we cast `$search` to an array the special handling in the else block won't be applied.